CHANGELOG.md

```diff
@@ -2,6 +2,10 @@
 All notable changes to this project will be documented in this file.
 
 ---
+## [1.5.1]
+* Fix:
+	* Fix duplicated target handler generated when getting commands from `ModernAVPlayerRemoteCommandFactory`
+
```

Comment:
add 

Breaking changes:
 - ModernAVPlayerRemoteCommandFactory become a class
 - Get most remote commands from lazy var instead of functions

Reply:
Done